db/models/reviews.model.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../../db/connection");
+
+exports.fetchReviews = () => {
+  return db
+    .query(
+      `SELECT owner, title, reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, designer, COUNT(comments.review_id)::INT as comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id GROUP BY reviews.review_id ORDER BY reviews.created_at DESC;`
+    )
+    .then((response) => {
+      if (response.rows.length === 0) {
+        return Promise.reject({ status: 404, message: "Path Not Found" });
```

Comment:
You already have a custom handler for 404, I don't believe there would be a way to have a 404 here without a bad path  (which you've handled) as this is not requesting by any particular id. If you take it out, does your test for 404 still pass?

Reply:
Yes it does! I will refactor. Thank you Verity!